integration_tests/datagen/ad_ctr/ad_ctr.go

```diff
@@ -55,7 +71,7 @@ type adCtrGen struct {
 func NewAdCtrGen() gen.LoadGenerator {
 	return &adCtrGen{
 		ctr:   make(map[int64]float64),
-		faker: gofakeit.New(0),
+		faker: gofakeit.New(1), // Use seed=1 to make it deterministic
```

Comment:
> Use fixed random seed so that I can use multiple datagen processes to generate multiple topics (a better approach is generating once and outputting to multiple sink, but that requires further refactoring on arguments)

I don't understand this. What's its relationship with "multiple datagen processes"? Besides, other `LoadGenerator` are still non-deterministic? I just don't know what you want to do :eyes:

Reply:
Oh, I got it. You want to 
```
./datagen --mode ad-ctr --topic ad_impression
./datagen --mode ad-ctr --topic ad_click
```
🥵

Reply:
It's a little weird, but I don't mind much. 😇

Reply:
Exactly. I've added this example to PR description. I agree it's a little bit weird